vinca/resolve.py

```diff
@@ -30,16 +30,11 @@ def resolve_pkgname_from_indexes(pkg_shortname, conda_index):
     for i in conda_index:
         if pkg_shortname in i:
             sys_platform = map_platform_python_to_conda[config.selected_platform]
-            if "conda-forge" in i[pkg_shortname].keys():
-                if sys_platform in i[pkg_shortname]["conda-forge"]:
-                    return i[pkg_shortname]["conda-forge"][sys_platform]
-                elif "unix" in i[pkg_shortname]["conda-forge"] and sys_platform in [
```

Comment:
I think the ability to use `unix` went missing here :)

Reply:
Deliberately! I don't think that rosdep can handle these cases, hence I removed it. In the ros-noetic PR I manually replaced all "unix" with "linux" + "osx" entries. There were only ~10.

Reply:
ok, fine! :)